docs/feature_flags.md

```diff
@@ -204,3 +204,13 @@ Enables ingestion of created timestamp. Created timestamps are injected as 0 val
 Currently Prometheus supports created timestamps only on the traditional Prometheus Protobuf protocol (WIP for other protocols). As a result, when enabling this feature, the Prometheus protobuf scrape protocol will be prioritized (See `scrape_config.scrape_protocols` settings for more details).
 
 Besides enabling this feature in Prometheus, created timestamps need to be exposed by the application being scraped.
+
+
+## Metadata Storage
+`--enable-features=metadata-storage`
```

Comment:
```suggestion

`--enable-features=metadata-storage`
```

Reply:
Maybe "metadata-in-wal"? unless we also store metadata in the TSDB block (I doubt)